mynaparrot/plugNmeet-client
# [DIGIS-521] - Changed the end screen.

### Title -[DIGIS-521] - Changed the end screen.

### PR for

**Before change** - Session end screen show "Room disconnected Code : ROOM_END".

**After change** - Session end screen show "Thank you! Digis" 

### Testing steps,
1. Login to DKYC dev-environment and create a session.
2. Start the new newly created session.
3. End the session and check the end screen.

### Acceptable criteria,
The session end screen should show "Thank you! Digis" 

### Reviewers,
@SandunRathsara @kalanainme @ThamalDilanka @IsuruSandaruwan97 @prashani97
